internal/runners/push/push.go

```diff
@@ -232,6 +233,12 @@ func (r *Push) Run(params PushParams) error {
 		}
 	}
 
+	// We have sucessfully pushed, if the project was initially headless we update the config
+	if intend&pushFromHeadless > 0 {
```

Comment:
This conditional isn't necessary. It doesn't hurt to clean the project mapping even when it's redundant.

Reply:
This should still be addressed.